index.bs

```diff
@@ -855,6 +863,26 @@ these steps.
    requested but not satisfied, then the value is `"default"`
    The getter steps are to return [=this=]'s {{[[CongestionControl]]}}.
 
+: <dfn for="WebTransport" attribute>rateControlFeedbackMinInterval</dfn>
+:: The application's preference, if changed from the default,
+   for the minimum interval between any two times when
```

Comment:
As mentioned above, I think we should make the default readable instead to simplify, which would mean rephrasing this.

Reply:
OK, I reworded it so the default is readable.